src/super_gradients/common/sg_loggers/metric_outputs.py

```diff
@@ -0,0 +1,38 @@
+from dataclasses import dataclass
+from typing import Union, Dict
+
+import numpy as np
+import torch
+
+
+@dataclass
+class PlottableMetricOutput:
```

Comment:
Thanks for making the changes we talked about 🙏 
I feel we are still not quite there yet tho) So my take on this class is this:

```
from typing import TypeAlias

@dataclass
class PlottableMetricOutput:
    """Metric output that can is represented as a graph.
    :attr image:    The image of the graph representing the metric value. 
                    This image will be logged as an image.
    """

    image: np.ndarray

ScalarOutput: TypeAlias = Union[float, int, torch.Tensor]

MetricOutput = Union[PlottableMetricOutput, ScalarOutput]
```

Imagine a case when you want to generate confusion matrix. What `scalar` value would correspond here?

So we should not be storing anything but `image` in `PlottableMetricOutput`. If we want to log area under curve for some reason - this should be logged as an additional value to the metric dict.

If we change the dataclass to suggested above, the `get_scalar_metric_output` can go.

Reply:
Changed here https://github.com/Deci-AI/super-gradients/commit/42ea92e2300738d1e4915d623fa46b52305b4821